hexrd/ui/material_structure_editor.py

```diff
@@ -291,8 +303,8 @@ def coords_equal(v1, v2, tol=1.e-5):
                 all(abs(x - y) < tol for x, y in zip(v1, v2))
             )
 
-        info_array = mat._atominfo
-        type_array = mat._atomtype
+        pos_array = mat.atom_pos
+        type_array = mat.atom_type
         charge_array = mat.charge
```

Comment:
The charge also needs to be synced. That's part of PR https://github.com/HEXRD/hexrd/pull/450

Reply:
Sounds good, thank you.